model/host/host.go

```diff
@@ -121,18 +121,22 @@ type Host struct {
 
 type HostGroup []Host
 
-// DockerOptions contains options for starting a container from host create
+// DockerOptions contains options for starting a container
 type DockerOptions struct {
 	// Optional parameters to define a registry name and authentication
 	RegistryName     string `bson:"registry_name,omitempty" json:"registry_name,omitempty"`
```

Comment:
Since you're going to be using mapstructure to populate this struct, tags should be set on all of them. Mapstructure will assume that the lowercase version of the field name matches the key if it's not specified, but it's good to be explicit

Reply:
good catch!